src/components/kpi/kpi.directive.coffee

```diff
@@ -112,8 +112,17 @@ angular
           ImpacKpisSvc.validateKpiTargets($scope.targets)
 
         $scope.hasContent = ->
-          return true if $scope.isEditing()
-          $scope.kpi && $scope.kpi.layout && $scope.kpi.data
+          !!($scope.kpi && $scope.kpi.layout && $scope.kpi.data)
+
+        $scope.showKpiContent = ->
+          # Newly added kpis start in edit mode (draft) and will have "no content".
+          !$scope.isLoading() && ($scope.hasContent() || $scope.kpi.isDraft)
```

Comment:
The logic in `isDataNotFound` is slightly different... using an `&&` rather than a `||`. 
```
$scope.isDataNotFound = ->
          !$scope.hasContent() && !$scope.kpi.isDraft
```